LocalWiki.php

```diff
@@ -139,6 +139,53 @@ function onBeforePageDisplay( OutputPage $out ) {
 	case 'loginwiki':
 		wfLoadExtension( 'GlobalWatchlist' );
 
+		break;
+	case 'metawiki':
+		$wgContactConfig['requestaccount'] = [
+			'RecipientUser' => 'Miraheze CVT',
+			'SenderName' => 'Account Creation Request Form via Meta',
+			'RequireDetails' => true,
+			'IncludeIP' => false,
+			'MustBeLoggedIn' => false,
+			'AdditionalFields' => [
+				'SelectIssue' => [
+					'label-message' => 'contactpage-requestaccount-selectissue',
```

Comment:
This and all following messages exist?

Reply:
They will be created on Meta once this is merged